pyproject.toml

```diff
@@ -94,23 +94,16 @@ filterwarnings = [
 ]
 
 [tool.ruff]
-select = [
-    "E",
-    "F", # flake8
+src = ["src"]
+lint.extend-select = [
     "D", # pydocstyle
 ]
-extend-ignore = [
+lint.ignore = [
```

Comment:
Conversely, `extend-ignore` was exactly the same as `ignore`, and is deprecated. (and everything lint-specific is supposed to be in a `lint` subsection now since Ruff is also a formatter)